controller.inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class ObjController {
+
+  function ObjController() {} //constructor
```

Comment:
You will see it with us but there are some code standarts, called [PSR](https://www.php-fig.org/psr/) for PHP Standards Recommendations. It prescribed to avoid truncated names or abbreviations. In this case Controller is enough. 
Don't need to learn them, we've got tools to automatically check for us.